label_sync/labels.yaml

```diff
@@ -1227,6 +1227,12 @@ repos:
         addedBy: anyone
   kubernetes-sigs/cluster-api:
     labels:
+      - color: c7def8
+        description: Categorizes issue or PR as related to a proposal.
```

Comment:
```suggestion
        description: Issues or PRs related to proposals
```

Reply:
I can change it, although the other kind/* labels have this style of description